cmd/soroban-cli/src/fee.rs

```diff
@@ -24,6 +24,8 @@ pub struct Args {
     /// Build the transaction and only write the base64 xdr to stdout
     #[arg(long, help_heading = HEADING_RPC)]
     pub build_only: bool,
+    #[arg(long, visible_alias = "seq-num", help_heading = HEADING_RPC)]
```

Comment:
Missing description on the sequence number (should also mention that it's optional)